content/200-orm/050-overview/500-databases/300-postgresql.mdx

```diff
@@ -8,6 +8,14 @@ The PostgreSQL data source connector connects Prisma ORM to a [PostgreSQL](https
 
 By default, the PostgreSQL connector contains a database driver responsible for connecting to your database. You can use a [driver adapter](/orm/overview/databases/database-drivers#driver-adapters) (Preview) to connect to your database using a JavaScript database driver from Prisma Client.
 
+:::info
+
+Need a Postgres database yesterday? Try [Prisma Postgres](https://www.prisma.io/postgres?utm_source=docs&utm_campaign=postgresql)! It's a lightning fast database with a generous free tier and pay-as-you-go pricing. [Visit the Console](https://console.prisma.io?utm_source=docs&utm_campaign=postgresql) to get started today.
```

Comment:
Thanks for the quick addition!

Is it maybe too marketing-y for a docs page? :) Perhaps touting the features and the unikernel/bare-metal architecture would fit better to the target audience and intent of this page? Just making sure our docs don't become too sales-y.

Reply:
```suggestion
Need a Postgres instance yesterday? 

With [Prisma Postgres](https://www.prisma.io/postgres?utm_source=docs&utm_campaign=postgresql) you can get a database running on bare-metal in three clicks. Connection pooling, real-time events, and automated backups are all included. [Visit the Console](https://console.prisma.io?utm_source=docs&utm_campaign=postgresql) to get started today.
```